README.md

```diff
@@ -30,57 +30,25 @@ Additional feature of the plugin that fulfills Invoicing domain is the ability t
     ```bash
     composer require sylius/invoicing-plugin
     ```
-
-2. Import configuration:
-
-    ```yaml
-    imports:
-        - { resource: "@SyliusInvoicingPlugin/Resources/config/config.yml" }
-    ```
-
-3. Import routing:
-
-    ```yaml
-    sylius_invoicing_plugin_admin:
-        resource: "@SyliusInvoicingPlugin/Resources/config/app/routing/admin_invoicing.yml"
-        prefix: /admin
     
-    sylius_invoicing_plugin_shop:
-        resource: "@SyliusInvoicingPlugin/Resources/config/app/routing/shop_invoicing.yml"
-        prefix: /{_locale}
-        requirements:
-            _locale: ^[a-z]{2}(?:_[A-Z]{2})?$
-    ```
+    > Remember to allow community recipes with `composer config extra.symfony.allow-contrib true` or during plugin installation process
 
-4. Add plugin class to your `AppKernel`:
+2. Copy plugin migrations to your migrations directory (e.g. `src/Migrations`) and apply them to your database:
 
-    ```php
-    $bundles = [
-        new \Knp\Bundle\SnappyBundle\KnpSnappyBundle(),
-        new \Prooph\Bundle\ServiceBus\ProophServiceBusBundle(),
-        new \Sylius\InvoicingPlugin\SyliusInvoicingPlugin(),
-    ];
+    ```bash
+    cp -R vendor/sylius/invoicing-plugin/migrations/* src/Migrations
+    bin/console doctrine:migrations:migrate
     ```
 
-5. Check if you have `wkhtmltopdf` binary. If not, you can download it [here](https://wkhtmltopdf.org/downloads.html).
```

Comment:
Shouldn't we still inform that such binary is needed to make InvoicingPlugin work properly and it's path is placed in `.env` file?

Reply:
That was also my intention... But I'm not 100% sure as well 😄

Reply:
Right, I forgot that knp-snappy-bundle is also powered by Symfony Flex :D